bootstrap-gruntwork-installer.sh

```diff
@@ -156,6 +162,7 @@ function install_fetch {
   local -r install_path="$1"
   local -r version="$2"
   local -r no_sudo="$3"
+  local -r retry_number="$4"
```

Comment:
Are these plumbed through for future usage?

Reply:
Yes, this now passes through correctly as verified by CLI invocation (see other commits)